WINDOWS_INSTRUCTIONS.md

```diff
@@ -0,0 +1,147 @@
+# Setting Up the Project on Windows
+
+Detailed Instructions for setting up this project on windows
+
+## Versions
+<ul>
+<li>Ruby Version: ruby-2.5.1</li>
+ <li>Rails Version: Rails 5.1.6</li>
+ <li>PostgreSQL Version: 9.5</li>
+ <li>MYSQL Server</li>
+ </ul>
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+<ul>
+<li>git-bash : [Git Bash](https://git-scm.com/)</li>
+<li>Ruby  :    [Rails Installer ](https://s3.amazonaws.com/railsinstaller/Windows/railsinstaller-3.4.0.exe)</li>
+<li>PostgreSQL : [PostgreSQL](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)</li>
+<li>Mysql Server : [Xampp](https://www.apachefriends.org/download.html)</li>
+  </ul>
+
+###### Packages Included in the rails Installer:

```

Comment:
Why is this heading 6th level directly from 3rd level?

If you only want to visually bold a piece of text, Use `**` rather than ######.